src/diy/smol-slime.md

```diff
@@ -12,38 +12,38 @@ Interested, have questions, or issues with this project? Chat with us in ***#smo
+| eByte Dongle (E104-BT5040U)                     | The most affordable option featuring a PCB antenna. It is available on AliExpress, with free shipping. <br /> It can also be found on Alibaba by `Chengdu Ebyte Electronic Technology Co., Ltd.` <br/> - `E104-BT5040U` is fully compatible with the Nordic Semiconductor nRF52840 Dongle. <br/> - The `E104-BT5032U` and `E104-BT5040UA` models, however, are not compatible. They are only capable of capturing BLE4.2 and BLE5.0 protocol packets. |
+| Nordic Semiconductor nRF52840 Dongle (PCA10059) | Official Nordic development hardware. They are available for purchase on Digikey or Mouser.                                                                                                                                                                                                                                                                                                         |
 
 ### Microcontroller Boards
-These boards use antenna designs that are not optimized for range. If you are using the same boards for both trackers and receiver, they will likely perform poorly without modification.
+These boards utilize antenna designs that are not optimized for range. If you are using the same boards for both trackers and receiver, they will likely perform poorly without modifications.
 
 | Board                      | Description                                                                                                                                            |                                                                                                                                         |
 | -------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------- |
-| SuperMini nRF52840         | Cheapest option overall. It is a clone of the nice!nano board. It is available on AliExpress with `compatible with nice!nano` or `Pro Micro` branding.<br>If you want to improve signal strength, you can replace the built-in antenna with a 31 mm wire. This creates a basic monopole antenna. |
-| Seeed Studio XIAO nRF52840 | Compact board, available on Seeed Studio.                                                                                                                                                                                                                                                        |
+| SuperMini nRF52840         | The most affordable option overall. It is a clone of the nice!nano board. It is available on AliExpress with `compatible with nice!nano` or `Pro Micro` branding.<br>If you wish to enhance signal strength, you can replace the built-in antenna with a 30.5 mm wire, which will create a basic monopole antenna. |
```

Comment:
i believe a 30.7mm wire is closer to the center of the 2.4ghz band used by nRF, if you want to have decimal precision